docs/RelayServer.md

```diff
@@ -0,0 +1,240 @@
+command should be enough to do this for an ubuntu based system.
+
+```
+sudo apt install build-essential python3-dev python3-venv pkg-config \
+     libssl-dev automake autoconf libtool libgraphviz-dev git libpq-dev
+```
+
+The installation instructions assume you create a dedicated user account and put
+files directly into the users home directory. The paths should be adapted to
+your needs.
+
+### parity/geth
+
+You need to run a parity or geth node, which provides the JSONRPC API to the
+relay server and the indexer. At the moment we recommend using parity together
+with the kovan chain.
+
+Please follow the [official documentation](https://wiki.parity.io/Setup) in
+order to install parity. Paritytech provides [binary
+releases](https://github.com/paritytech/parity-ethereum/releases). If you're
+running ubuntu, the following command allows you to install parity:
+
+```
+curl -O https://releases.parity.io/v1.11.7/x86_64-unknown-linux-gnu/parity_1.11.7_ubuntu_amd64.deb
+sudo dpkg -i parity_1.11.7_ubuntu_amd64.deb
+```
+
+The relay server needs an unlocked account. Therefore, make sure you're not
+exposing the JSONRPC endpoint. By default parity listens on the local interface
+for new connections, so any user with a login on the machine could also access
+the JSONRPC endpoint.
+
+<!-- XXX Explain how to create an account -->
+
+The following command line starts parity with the kovan chain. Please adapt the
+--author, --unlock and --password arguments
+
+```
+parity --no-warp --auto-update none --no-download --chain kovan \
+    --author 0x987654321 --unlock 0x987654321 --password /path/to/password-file \
+    --jsonrpc-hosts all
+```
+
+One command line option we like to highlight is the `--no-warp` option. If you
+don't specify this option, parity will enable 'Warp Synchronization'. Warp
+synchronization allows for a faster synchronization, but the relay server will
+not work reliably with warp synchronization turned on. The [offical
+documentation](https://wiki.parity.io/Getting-Synced#warp-synchronization)
+contains a description of the behaviour that is problematic:
+
+> Note, at present, snapshotting does not place all of the block or receipt data
+> into the database. This means you will not get information relating to
+> transactions more than a few days old.
+
+So, please double check that warp synchronization is turned off when using
+parity. The whole trustlines system may appear to work, but will give wrong
+results if you run with warp mode enabled.
+
+### contracts
+The [trustlines-contracts
+repository](https://github.com/trustlines-network/contracts) contains the
+solidity contracts to be deployed on the blockchain. The ('how to deploy the
+contracts guide')[] <!-- XXX --> contains more information on how to deploy the
+contracts. We assume from now on that the contracts have already been deployed
+and that the `addresses.json` file has been copied to the users home directory.
```

Comment:
This file is not generated when using the command line tool for normal deploying. Usually you will not know all currency networks at the same time. Right now you have to write it manually.

Reply:
Can you please write a short description then? I haven't read the documentation for the contract deployment CLI tools yet.

Reply:
The deploy tool will return the address of the deployed contract. You should describe here how the address.json file is structured and can be written manually if the addresses of the currency network, exchanges, and unwrapping ether contract are known. 
Do you think we need a tool to generate the file?

Reply:
I don't think we need a tool at the moment. Can you please write that description? I think it's easier for you and it doesn't make sense to just describe the json format. We should document how that relates to the contract deployment CLI tools.

Reply:
Ok, will do that tomorrow.